src/components/mark/_index.scss

```diff
@@ -0,0 +1 @@
+@import 'mark.module';
```

Comment:
`.module` just so babel & webpack can distinguish which files to process as modules.